pyteal/ast/router.py

```diff
@@ -176,26 +173,27 @@ def is_empty(self) -> bool:
 OnCompleteAction.__module__ = "pyteal"
 
 
-@dataclass(frozen=True)
 class BareCallActions:
```

Comment:
Made this into a regular class because I found this to be the easiest way to allow one mutable field (`stack_frames`) while maintaining all the others as "immutable".